cli.js

```diff
@@ -39,15 +39,10 @@ const searchForScript = (basedir, scriptPath) => {
         triedPaths.push(tryPath)
       }
     })
-    reject(triedPaths)
+    throw new Error(`Could not find "${scriptPath}" in:\n\n${triedPaths.join('\n')}\n`)
   });
 }
 
-const throwSearchFail = (script, triedPaths) => {
-  console.log(triedPaths)
-  throw new Error(`Could not find "${script}" in:\n\n${triedPaths.join('\n')}\n`)
-}
-
```

Comment:
No reason to have this separate.  It was only used as the reject case of the search function.  Moved into the search function reject place.